dgisim/src/dices.py

```diff
@@ -226,6 +197,34 @@ class ActualDices(Dices):
         for elem, i in _LEGAL_ELEMS_ORDERED_DICT.items()
     }
 
+    # higher PRIORITY - should be spent first
+    # _ELEMENTS_BY_DECREASING_GLOBAL_PRIORITY: tuple[Element, ...] = (
+    #     Element.PYRO,
+    #     Element.HYDRO,
+    #     Element.ANEMO,
+    #     Element.ELECTRO,
+    #     Element.DENDRO,
+    #     Element.CRYO,
+    #     Element.GEO,
+    # )
+    _ELEMENTS_BY_DECREASING_GLOBAL_PRIORITY: tuple[Element, ...] = _LEGAL_ELEMS_ORDERED[1:][::-1]
+    # probably correct priorities
```

Comment:
should this line of comment be moved one line up?

Reply:
done